packages/wallets/trezor/src/adapter.ts

```diff
@@ -0,0 +1,169 @@
+import type { WalletName } from '@solana/wallet-adapter-base';
+import TrezorConnect, { DEVICE, DEVICE_EVENT } from '@trezor/connect-web';
```

Comment:
These will need to be imported as types only. External SDKs must be loaded async on connect.

Reply:
https://github.com/solana-labs/wallet-adapter/pull/839/commits/12f08375621bd1d53c4b73bd1eecfa121ddf23bd